src/main/java/io/games/simple/webapp/controller/LoginController.java

```diff
@@ -1,36 +1,13 @@
 package io.games.simple.webapp.controller;
 
-
-import io.games.simple.webapp.entity.UserEntity;
-import io.games.simple.webapp.service.UserService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.ResponseBody;
 
 @Controller
 public class LoginController {
 
-    @GetMapping("/login")
-    public String logInPage(){
+    @GetMapping("/login") public String renderLogin(){
         return "login";
     }
 
-    //Code below to erase. It purpose is to fast add test user to DB
```

Comment:
Maybe we should add tags to comments like "ERASE: " so that we can later do ctrl + f and just go through them easily.

Reply:
Grat idea Karol.